### PR TITLE
i18n(es): update of  `astro-pages.mdx`, `markdown-content.mdx`, `rss.mdx`, `styling.mdx` & `auto.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Multiple updates of pages based on the following info:

-astro-pages.mdx → 1d0823fab64c8e2601c3347309efd4ee78a221de

-markdown-content.mdx → 1d0823fab64c8e2601c3347309efd4ee78a221de

-rss.mdx → d373149aee53cf21fb4530034bfaa9f916a9d02f & 3ff3fa2a4acd8366f529040ed539ef8022bdfd9e

-styling.mdx → 1d0823fab64c8e2601c3347309efd4ee78a221de

-auto.mdx → e49d2de03f0b044e2dbd19a9b04476c4a4d63158

### CODE REVIEW

File: src/pages/es/guides/styling.mdx [hunk 1]
```diff
@@ -478,6 +478,13 @@ Vue en Astro es compatible con los mismos métodos que `vue-loader`:
 
 Svelte en Astro también funciona exactamente como se espera: [Svelte Styling Docs][svelte-style].
 
+## Estilando Markdown
+
+Cualquier método de estilo de Astro está disponible para un [componente de diseño de Markdown](/es/core-concepts/layouts/#plantillas-de-markdownmdx).
+
+You can apply global styles to your Markdown content by adding [`<style>` tags](#estilos-globales) and [imported stylesheets](#estilos-externos) to the layout that wraps your page content. You can also add [CSS integrations](#integraciones-css) including [Tailwind](/es/guides/integrations-guide/tailwind/).
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/styling.mdx:485]:
```suggestion
Puede aplicar estilos globales a tu contenido de Markdown agregando [etiquetas `<style>`](#estilos-globales) y [hojas de estilo importadas](#estilos-externos) al la plantilla que envuelve el contenido de tu página. También puedes agregar [integraciones de CSS](#integraciones-css) incluyendo [Tailwind](/es/guides/integrations-guide/tailwind/).
```

File: src/pages/es/guides/styling.mdx [hunk 2]
```diff
@@ -478,6 +478,13 @@ Vue en Astro es compatible con los mismos métodos que `vue-loader`:
...
+
+If you are using Tailwind, the [typography plugin](https://tailwindcss.com/docs/typography-plugin) can be useful for styling Markdown.
```
Comment (kevinzunigacuellar) [inline src/pages/es/guides/styling.mdx:487]:
```suggestion
Si usas Tailwind, el [plugin de tipografía] (https://tailwindcss.com/docs/typography-plugin) puede ser útil para estilar Markdown.
```

File: src/pages/es/install/auto.mdx [hunk 3]
```diff
@@ -89,26 +89,40 @@ Si no es posible abrir el proyecto en el navegador, regresa a la terminal donde
 
 ## Plantillas de Inicio
 
-También comenzar un proyecto nuevo de Astro basado en un repositorio de GitHub existente pasando el argumento `--template` al comando `create astro`.
+Puedes empezar un nuevo proyecto de Astro basado en un [ejemplo oficial](https://github.com/withastro/astro/tree/main/examples) o en la rama `main` de cualquier repositorio de GitHub pasando un argumento `--template` al comando `create astro`.
 
 <PackageManagerTabs>
   <Fragment slot="npm">
   ```shell
+  # create a new project with an official example
```
Comment (kevinzunigacuellar) [inline src/pages/es/install/auto.mdx:97]:
```suggestion
  # crea un nuevo proyecto a partir de un ejemplo oficial
```

File: src/pages/es/install/auto.mdx [hunk 4]
```diff
@@ -89,26 +89,40 @@ Si no es posible abrir el proyecto en el navegador, regresa a la terminal donde
...
+  npm create astro@latest -- --template <example-name>
+
+  # create a new project based on a GitHub repository’s main branch
```
Comment (kevinzunigacuellar) [inline src/pages/es/install/auto.mdx:100]:
```suggestion
  # crea un nuevo proyecto basado en la rama `main` de un repositorio de GitHub
```

File: src/pages/es/install/auto.mdx [hunk 5]
```diff
@@ -89,26 +89,40 @@ Si no es posible abrir el proyecto en el navegador, regresa a la terminal donde
...
+  # create a new project with an official example
+  npm create astro@latest -- --template <example-name>
+
+  # create a new project based on a GitHub repository’s main branch  
   npm create astro@latest -- --template <github-username>/<github-repo>
   ```
   </Fragment>
   <Fragment slot="pnpm">
   ```shell
+  # create a new project with an official example
```
Comment (kevinzunigacuellar) [inline src/pages/es/install/auto.mdx:106]:
```suggestion
  # crea un nuevo proyecto a partir de un ejemplo oficial
```

File: src/pages/es/install/auto.mdx [hunk 6]
```diff
@@ -89,26 +89,40 @@ Si no es posible abrir el proyecto en el navegador, regresa a la terminal donde
...
+  pnpm create astro@latest --template <example-name>
+  
+  # create a new project based on a GitHub repository’s main branch
```
Comment (kevinzunigacuellar) [inline src/pages/es/install/auto.mdx:109]:
```suggestion
  # crea un nuevo proyecto basado en la rama `main` de un repositorio de GitHub
```

File: src/pages/es/install/auto.mdx [hunk 7]
```diff
@@ -89,26 +89,40 @@ Si no es posible abrir el proyecto en el navegador, regresa a la terminal donde
...
   pnpm create astro@latest --template <github-username>/<github-repo>
   ```
   </Fragment>
   <Fragment slot="yarn">
   ```shell
+  # create a new project with an official example
+  yarn create astro --template <example-name>
+  
+  # create a new project based on a GitHub repository’s main branch
```
Comment (kevinzunigacuellar) [inline src/pages/es/install/auto.mdx:118]:
```suggestion
  # crea un nuevo proyecto basado en la rama `main` de un repositorio de GitHub
```

File: src/pages/es/install/auto.mdx [hunk 8]
```diff
@@ -89,26 +89,40 @@ Si no es posible abrir el proyecto en el navegador, regresa a la terminal donde
...
+  # create a new project with an official example
+  pnpm create astro@latest --template <example-name>
+  
+  # create a new project based on a GitHub repository’s main branch
   pnpm create astro@latest --template <github-username>/<github-repo>
   ```
   </Fragment>
   <Fragment slot="yarn">
   ```shell
+  # create a new project with an official example
```
Comment (kevinzunigacuellar) [inline src/pages/es/install/auto.mdx:115]:
```suggestion
  # crea un nuevo proyecto a partir de un ejemplo oficial
```

**Review by kevinzunigacuellar — COMMENTED** [review]
gracias @dreyfus92! Deje unas sugerencias 💜

File: src/pages/es/guides/rss.mdx [hunk 9]
```diff
@@ -36,114 +36,153 @@ Primero, instala `@astrojs/rss` utilizando tu gestor de paquetes favorito:
...
+      link: `/blog/${post.slug}/`,
+    })),
+  });
+}
+```
 
-```js
-// src/pages/rss.xml.js
-import rss from '@astrojs/rss';
+Puedes configurar tu esquema de colección para forzar estas propiedades RSS esperadas. Importa y aplica `rssSchema` para asegurarte que cada entrada de colección produzca un item RSS válido.
```
[the remark this answers is not in the document]
Comment (dreyfus92) [reply src/pages/es/guides/rss.mdx:94]: Enforzar no existe en español, es un anglicismo adecuado de `enforce`, no se si sería mejor poner "ejecutar". ¿Qué opinas?

File: src/pages/es/core-concepts/astro-pages.mdx [hunk 10]
```diff
@@ -22,6 +22,8 @@ Astro es compatible con los siguientes tipos de archivos en el directorio `src/p
 
 Astro aprovecha una estrategia de enrutamiento llamada **enrutamiento basado en archivos**. Cada archivo `.astro` en la carpeta `src/pages` se convierte en una página o un endpoint en tu proyecto de acuerdo a su ruta.
 
+Un archivo puede generar múltiples páginas usando [enrutamiento dinámico](/es/core-concepts/routing/#rutas-dinámicas). Esto te permite crear páginas incluso si tu contenido está fuera del directorio especial `/pages/`, como en una [colección de contenido](/es/guides/content-collections/#rendering-entry-content) o un [CMS](/es/guides/cms/).
```
Comment (kevinzunigacuellar) [inline src/pages/es/core-concepts/astro-pages.mdx:25]:
```suggestion
Un archivo puede generar múltiples páginas usando [enrutamiento dinámico](/es/core-concepts/routing/#rutas-dinámicas). Esto te permite crear páginas incluso si tu contenido está fuera del directorio especial `/pages/`, como en una [colección de contenido](/es/guides/content-collections/) o un [CMS](/es/guides/cms/).
```

**Pull request merged**